homeassistant/components/toon/sensor.py

```diff
@@ -124,6 +128,11 @@ def __init__(self, coordinator: ToonDataUpdateCoordinator, *, key: str) -> None:
             name=SENSOR_ENTITIES[key][ATTR_NAME],
         )
 
+        self._attr_last_reset = SENSOR_ENTITIES[key][ATTR_LAST_RESET]
+        self._attr_state_class = SENSOR_ENTITIES[key][ATTR_STATE_CLASS]
+        self._attr_unit_of_measurement = SENSOR_ENTITIES[key][ATTR_UNIT_OF_MEASUREMENT]
+        self._sttr_device_class = SENSOR_ENTITIES[key][ATTR_DEVICE_CLASS]
```

Comment:
If you change these to use `.get`, you can drop all the entries where you set an attribute to `None` from your code above `ATTR_LAST_RESET: None`

Reply:
(this could be done in a future PR)